specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/multivariate/models.cadl

```diff
@@ -0,0 +1,370 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+
+using Cadl.Http;
+using Cadl.Rest;
+using Azure.Core;
+
+namespace AnomalyDetector.Multivariate;
+
+enum FillNAMethodKnownValues {
+  "Previous",
+  "Subsequent",
+  "Linear",
+  "Zero",
+  "Fixed",
+}
+
+@doc("An optional field, indicating how missing values will be filled. One of Previous, Subsequent, Linear, Zero, Fixed.")
+@knownValues(FillNAMethodKnownValues)
+model FillNAMethod is string;
+
+enum MultivariateDetectionStatus {
+  Created: "CREATED",
+  Running: "RUNNING",
+  Ready: "READY",
+  Failed: "FAILED",
+}
+
+@doc("Data schema of input data source: OneTable or MultiTable. The default DataSchema is OneTable.")
+enum DataSchema {
+  @doc("OneTable means that your input data are all in one CSV file, which contains one 'timestamp' column and several variable columns. The default DataSchema is OneTable.")
+  "OneTable",
+  @doc("MultiTable means that your input data are separated in multiple CSV files, in each file containing one 'timestamp' column and one 'variable' column, and the CSV file name should indicate the name of the variable. The default DataSchema is OneTable.")
+  "MultiTable",
+}
+
+enum AlignMode {
+  "Inner",
+  "Outer",
+}
```

Comment:
Java apiview: Should this be expandable string enum? Do we expect to have more modes in the future?

Same for "DataSchema" and "ModelStatus"

Reply:
Currently no plan for changing that/adding more modes and "modelstatus", but we'll have plan to add more data schema, if we change that, we might have to change the swagger again, I suggest we keep this as a note and refine to expandable string enum for next API iteration.  @weidongxu-microsoft

Reply:
enum means you cannot change the value in future, new api-version or not.

So, if you ever plan to add a value, use extensible enum now.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#enums--sdks-client-libraries

Reply:
@catalinaperalta and @lmazuel  do you think this change could happen in this version? or it'll need much effort and we could delay to next version?

Reply:
@jr-MS will all of the enums be extensible? Or if there's a specific list, please share the list with me and I can take a look at making them extensible.

Reply:
Only DataSchema

Reply:
done